### PR TITLE
feat(BreadcrumbList schema): Add support for BreadcrumbList schema

### PR DESCRIPTION
**Pull request by iamgarima**
The support is added for home, section and story pages

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
```
Comment (gja) [inline CHANGELOG.md]: how did changelog changes come into PR? That too a deployed one?

File: src/structured-data/structured-data-tags.js [hunk 2]
```diff
@@ -184,6 +186,51 @@ function generateWebSiteData(structuredData = {}, story = {}, publisherConfig =
...
+  function getCollectionPageCrumbs({ slug = "", name = "" } = {}) {
+    const url = `${domain}/collection/${slug}`;
+    return getSchemaListItem(2, name, url);
+  }
+
+  function getStoryPageCrumbs({ slug = "", headline = "", sections: storySections = [] } = {}) {
+    const currentSection = sections.filter(section => section.slug === storySections[0]["slug"])[0];
+    const sectionCrumbsList = getSectionPageSchema(currentSection);
+    const position = sectionCrumbsList.length + 2;
+    const url = `${domain}/${slug}`;
```
Comment (gja) [inline src/structured-data/structured-data-tags.js]: You should get story.url now

File: src/structured-data/structured-data-tags.js [hunk 3]
```diff
@@ -184,6 +186,51 @@ function generateWebSiteData(structuredData = {}, story = {}, publisherConfig =
...
+    const { name, slug } = parentSection;
+    const url = `${domain}/${slug}`;
+    crumbsList.push(getSchemaListItem(itemListOrder, name, url));
+    return addCrumb(crumbsList, parentSection, ++itemListOrder);
+  }
+
+  function getSectionPageCrumbs({ slug = "", name = "" } = {}) {
+    const url = `${domain}/${slug}`;
+    const crumbsList = [getSchemaListItem(2, name, url)];
+    const currentSection = sections.filter(section => section.slug === slug)[0];
```
Comment (gja) [inline src/structured-data/structured-data-tags.js]: use section.url now

File: src/structured-data/structured-data-tags.js [hunk 4]
```diff
@@ -89,7 +91,7 @@ function generateArticleData (structuredData = {}, story = {}, publisherConfig =
 
   return Object.assign({}, generateCommonData(structuredData, story, publisherConfig), {
     "author": authorData(authors),
-    "keywords": metaKeywords,
+    "keywords": metaKeywords.join(','),
```
Comment (sinujohn91) [inline src/structured-data/structured-data-tags.js]: you sure about this change?

File: src/structured-data/structured-data-tags.js [hunk 5]
```diff
@@ -184,13 +186,52 @@ function generateWebSiteData(structuredData = {}, story = {}, publisherConfig =
   return getSchemaWebsite(structuredData.website);
 }
 
+function generateBreadcrumbListData(pageType = "", publisherConfig = {}, data = {}) {
```
Comment (sinujohn91) [inline src/structured-data/structured-data-tags.js]: publisherConfig i think we generally refer to publisher.yml file...

File: src/structured-data/structured-data-tags.js [hunk 6]
```diff
@@ -184,13 +186,52 @@ function generateWebSiteData(structuredData = {}, story = {}, publisherConfig =
...
+  function getSectionPageCrumbs({ "section-url":sectionUrl = "", slug = "", name = "" } = {}) {
+    const crumbsDataList = [{ sectionUrl, name }];
+    const currentSection = sections.find(section => section.slug === slug);
+    addCrumb(crumbsDataList, currentSection);
+    return crumbsDataList.map(({sectionUrl = "", name = ""}, index) => getSchemaListItem(index + 2, name, sectionUrl));
+  }
+
+  function getStoryPageCrumbs({ headline = "", url = "", sections: storySections = [] } = {}) {
+    const storySectionSlug = get(storySections, [0, "slug"], "");
+    const currentSection = sections.find(section => section.slug === storySectionSlug);
```
Comment (sinujohn91) [inline src/structured-data/structured-data-tags.js]: this is already available.

File: src/structured-data/structured-data-tags.js [hunk 7]
```diff
@@ -184,13 +186,52 @@ function generateWebSiteData(structuredData = {}, story = {}, publisherConfig =
...
+    const sectionCrumbsList = getSectionPageCrumbs(currentSection);
+    const position = sectionCrumbsList.length + 2;
+    sectionCrumbsList.push(getSchemaListItem(position, headline, url));
```
Comment (sinujohn91) [inline src/structured-data/structured-data-tags.js]: again we should pass an array to getSchemaBreadcrumbList and that guy should worry about the position, and the schema.

File: src/structured-data/structured-data-tags.js (hunk 4 above)
Comment (iamgarima) [reply src/structured-data/structured-data-tags.js]: Yup :D

**Pull request merged**